em-app/src/csr.rs

```diff
@@ -4,20 +4,21 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use crate::Error;
```

Comment:
nit:

We haven't really set a rustfmt config for this repo (as far as I know), but for our internal repos we use the `StdExternalCrate` style for our imports (https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#group_imports)

Reply:
Thank you for the explanation, I will keep that in mind for styling!